x/supply/internal/keeper/integration_test.go

```diff
@@ -0,0 +1,32 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	keep "github.com/cosmos/cosmos-sdk/x/supply/internal/keeper"
+	"github.com/cosmos/cosmos-sdk/x/supply/internal/types"
+)
+
+var (
+	multiPerm  = "multiple permissions account"
+	randomPerm = "random permission"
+	holder     = "holder"
+)
+
+// nolint: deadcode unused
+func createTestApp(isCheckTx bool) (*simapp.SimApp, sdk.Context) {
+	app, ctx := simapp.Setup(isCheckTx)
```

Comment:
See above.

Reply:
this function uses the ctx returned, so I don't see a reason not to return the ctx as well. maybe I should update the function signature to `setupTestApp` or something?